packages/react-router-dev/vite/plugin.ts

```diff
@@ -1836,32 +1836,39 @@ async function handlePrerender(
     );
   }
 
-  async function prerenderData(
```

Comment:
This was accidentally a locally scoped function inside the parent `handlePreRender` function.  No changes were made - it just moves out to be a module level function